### PR TITLE
chore: task list filters [DET-5390]

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Continue the same work of migrating over to native antd filters for Task List page.

Before:
Screen Shot 2021-05-27 at 4 37 15 PM

After:
Screen Shot 2021-05-27 at 4 36 10 PM
Screen Shot 2021-05-27 at 4 36 19 PM

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/pages/TaskList.tsx [hunk 1]
```diff
@@ -190,24 +267,100 @@ const TaskList: React.FC = () => {
...
+      {
+        dataIndex: 'id',
+        key: 'id',
+        render: taskIdRenderer,
+        sorter: (a: CommandTask, b: CommandTask): number => alphanumericSorter(a.id, b.id),
+        title: 'Short ID',
+      },
+      {
+        filterDropdown: typeFilterDropdown,
+        filters: Object.values(CommandType)
```
Comment (apizzini) [inline webui/react/src/pages/TaskList.tsx:280]: What about adding type icons here?

Schermata 2021-06-01 alle 08 21 25

**Review by apizzini — APPROVED** [review]
LGTM 👍

File: webui/react/src/pages/TaskList.tsx (hunk 1 above)
Comment (hkang1) [reply webui/react/src/pages/TaskList.tsx:280]: great idea

Comment (hkang1) [reply webui/react/src/pages/TaskList.tsx]: Ended up having icons and labels together. With just the icons, it looked sparse and the filter dropdown had a minimum size required to house both the `Reset` and `Ok` buttons.

**Pull request merged**